readme.md

```diff
@@ -139,7 +180,15 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+var Instructor = function(name){
+  this.name = name;
+  this.receivePresent = function(gift){
+    console.log(name + " promptly drops the " + gift + " on the floor."); // You guys are so brutal XD
+  }
+};
+
+andy = new Instructor("Andy");
+andy.receivePresent("Resin Laying Deer Figurine, Gold");
 
 ```
 
```

Comment:
:+1:

Could also define `receivePresent` as a prototypal method since the method is identical for each `Instructor` (aside from the argument).